c7n/resources/org.py

```diff
@@ -191,6 +191,61 @@ def parse_query(self):
                 self.account_config["org-account-role"] = "AWSControlTowerExecution"
 
 
+@OrgUnit.filter_registry.register('org-unit')
+@OrgAccount.filter_registry.register('org-unit')
+class OrgUnitFilter(ValueFilter):
+    """Filter resources by their containment within an ou."""
```

Comment:
I used these policies when testing locally, might be helpful in a docstring.

```suggestion
    """Filter resources by their containment within an ou.

    .. code-block:: yaml

        policies:
          - name: org-units-by-parent-ou
            resource: aws.org-unit
            filters:
              - type: org-unit
                key: Name
                value: dev

          - name: org-accounts-by-parent-ou
            resource: aws.org-account
            filters:
              - type: org-unit
                key: Name
                value: dev
    """
```

Reply:
cool, fwiw tag filters on the ou also work.